app/code/Bulbulatory/Recommendations/Model/Table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Model;
+
+class Table extends \Magento\Framework\Model\AbstractModel implements \Magento\Framework\DataObject\IdentityInterface
+{
+    const CACHE_TAG = 'bulbulatory_recommendations_table';
+
+    protected $_cacheTag = 'bulbulatory_recommendations_table';
+    protected $_eventPrefix = 'bulbulatory_recommendations_table';
+
+    protected function _construct()
+    {
+        $this->_init('Bulbulatory\Recommendations\Model\ResourceModel\Table');
+    }
+
+    public function getIdentities()
+    {
+        return [self::CACHE_TAG . '_' . $this->getId() ];
+    }
+
+    public function getDefaultValues()
+    {
+        $values = [];
+        return $values;

```

Comment:
You don't need to declare a variable just to return it in the next line, simply returning an empty array will work as well.

Reply:
@TM18

> You don't need to declare a variable just to return it in the next line, simply returning an empty array will work as well.

Fixed!